src/client/cli/formatter/table_formatter.cpp

```diff
@@ -53,91 +53,150 @@ std::string to_usage(const std::string& usage, const std::string& total)
     return fmt::format("{} out of {}", mp::MemorySize{usage}.human_readable(), mp::MemorySize{total}.human_readable());
 }
 
-std::string generate_instance_info_report(const mp::InfoReply& reply)
+std::string generate_snapshot_details(const mp::DetailedInfoItem& item)
 {
     fmt::memory_buffer buf;
+    const auto& fundamentals = item.snapshot_info().fundamentals();
+
+    fmt::format_to(std::back_inserter(buf), "{:<16}{}\n", "Snapshot:", fundamentals.snapshot_name());
```

Comment:
The 16-char width results in 4 spaces of separation here, unlike the 3 that the spec prescribed. However, looking at the output, 4 it is actually a better choice, because it maintains alignment with instance info, so :+1:

I have updated the spec accordingly.